pages/sdk/signers.mdx

```diff
@@ -0,0 +1,13 @@
+# Signers
+
+Safe Smart Accounts can be setup with multiple and different signer accounts.
+
+The following guides show how to create a signer account with different service providers and use them to initialize any of the kits from the Safe\{Core\} SDK.
+
+- [Magic](./signers/magic.mdx)
+- [Privy](./signers/privy.mdx)
+- [Web3Auth](./signers/web3auth.mdx)
+
+## Resources
+
+- [Demo app on GitHub](https://github.com/5afe/safe-signers)
```

Comment:
In the README of the app's GitHub, can we add some instructions on starting the app, for example: which keys to get and where from, what values they should be assigned to, pnpm i, pnpm dev, etc.

The aim with this would be that the demo would also contain minimal standalone instructions (if discovered from Google, etc).

Reply:
Would also be helpful to add the Help and License section. A good example: https://github.com/5afe/safe-passkeys-tutorial